apis/validator/aggregate_attestation.yaml

```diff
@@ -7,6 +7,9 @@ get:
     A 503 error must be returned if the block identified by the response
     `beacon_block_root` is optimistic (i.e. the aggregated attestation attests
     to a block that has not been fully verified by an execution engine).
+
+    A 500 error must be returned if no attestation is available for the requested
+    `attestation_data_root`.
```

Comment:
a 500 would be super weird, why are we going with that?

Reply:
The sane alternative is a 404, which is not ideal either.

Reply:
It's more difinitive, and consistent with what we do when we have routes not found on the states endpoint... I'm happy with that.